addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -19,153 +17,242 @@
 
 <template id="index" name="Find Resellers">
     <t t-call="website_crm_partner_assign.layout">
-        <div class="col-lg-12">
-            <h1 class="text-center">
-                Looking For a Local Store?
-            </h1><h2 class="text-center text-muted">
-                Contact a reseller
-            </h2>
-        </div>
-
-        <div class="col-lg-3 mb32" id="partner_left">
-
-            <ul id="reseller_grades" class="nav nav-pills flex-column mt16">
-                <li class="nav-header nav-item"><h3>Filter by Level</h3></li>
-                <t t-foreach="grades" t-as="grade">
-                    <li class="nav-item">
+        <div class="o_wcrm_filters_top d-flex d-print-none align-items-center justify-content-end flex-wrap gap-2 w-100">
+            <h4 class="my-0 me-auto pe-sm-4">
+                Find a reseller <t t-foreach="countries" t-as="country"><span t-if="country['active'] and country['country_id'][0] != 0">in <t t-out="country['country_id'][1]"/></span></t>
```

Comment:
I'm not sure it's actually possible for it to be not set but since we check whether `country['country_id']` is truthy everywhere we should probably do it here too.

Reply:
Thanks a lot for the review ! 

Btw, @tde-banana-odoo said your team will get on this PR after the freeze but, after this review, do you think it's possible to merge before ? If it's not possible, I will switch my focus on other tasks and wait for you, just let me know 👍